src/OptionsCompounder.sol

```diff
@@ -11,16 +11,16 @@ import {IOptionsToken} from "optionsToken/src/interfaces/IOptionsToken.sol";
 import {ReaperAccessControl} from "vault-v2/mixins/ReaperAccessControl.sol";
 import {ISwapperSwaps, MinAmountOutData, MinAmountOutKind} from "vault-v2/ReaperSwapper.sol";
 import {IERC20} from "oz/token/ERC20/IERC20.sol";
-import {ERC20} from "oz/token/ERC20/ERC20.sol";
 import {Initializable} from "oz-upgradeable/proxy/utils/Initializable.sol";
 import {IOracle} from "optionsToken/src/interfaces/IOracle.sol";
 import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
 
 enum ExchangeType {
     UniV2,
     Bal,
+    UniV3,
     VeloSolid,
```

Comment:
do we for sure want to keep this -- we could only have one or the other, or we could put same enum in both op an arb-mnt-bsc branch

Reply:
I see, there is such type already in ReaperBaseStrategyv4, it would require to adjust this contract also for arb-mnt-bsc branch